lib/zoho_sign/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module ZohoSign
-  VERSION = "0.1.1"
+  VERSION = "0.1.2"
```

Comment:
Please revert this. I will handle it myself once is merged. Thank you

```suggestion
  VERSION = "0.1.1"
```